__tests__/shared/playwright_fixtures.js

```diff
@@ -0,0 +1,272 @@
+        await page.goto('/reset');
+        await page.waitForURL(/\/reset/);
+        await use(page);
+    },
+
+    resetPasswordPage: async ({ page }, use) => {
+        await page.goto('/reset/password');
+        await page.waitForURL(/\/reset\/password/);
+        await use(page);
+    },
+
+    /*************************
+     * AUTHENTICATED SESSION *
+     *************************/
+    loggedInPageStudent: async ({ page }, use) => {
+        await page.goto('/login');
+        await page.fill('input[name="email"]', 'studentuser@open-response.org');
+        await page.fill('input[name="rawPassword"]', 'studentstudent');
+        const log_in_button = page.locator('input[type="submit"][value="Log in"]');
+        await log_in_button.click();        
+        await page.waitForURL('/');
+        await use(page);
+    },
+
+    loggedInPageInstructor: async ({ page }, use) => {
+        await page.goto('/login');
+        await page.fill('input[name="email"]', 'teacheruser@open-response.org');
+        await page.fill('input[name="rawPassword"]', 'teacherteacher');
+        const log_in_button = page.locator('input[type="submit"][value="Log in"]');
+        await log_in_button.click();
+        await page.waitForURL('/');
+        await use(page);
+    },
+
+    // Does not matter if the user is a student or instructor
+    profilePage: async ({ loggedInPageStudent }, use) => {
+        await loggedInPageStudent.goto('/profile');
+        await loggedInPageStudent.waitForURL(/\/profile/);
+        await use(loggedInPageStudent);
+    },
+
+    // Does not matter if the user is a student or instructor
+    profileEditPage: async ({ profilePage }, use) => {
+        const editProfileButton = profilePage.locator('div.profileButton >> button');
+        await editProfileButton.click();
+        await profilePage.waitForSelector('div.profileButton');
+        await profilePage.getByText('Save').click();
+        await use(profilePage);
+    },
+
+    // Does not matter if the user is a student or instructor
+    confirmPage: async ({ loggedInPageStudent }, use) => {
+        await loggedInPageStudent.goto('/confirm');
+        await loggedInPageStudent.waitForURL(/\/confirm/);
+        await use(loggedInPageStudent);
+    },
+
+    //! does not matter if the user is a student or instructor - IT SHOULD MATTER
+    createCoursePage: async ({ loggedInPageInstructor }, use) => {
+        await loggedInPageInstructor.goto('/createCourse');
+        await loggedInPageInstructor.waitForURL(/\/createCourse/);
+        await use(loggedInPageInstructor);
+    },
+
+
+    joinCourseModalStudent: async ({ loggedInPageStudent }, use) => {
+        await loggedInPageStudent.locator('button.join-course').click();
+        await loggedInPageStudent.waitForSelector('div.centeredModal');
+        await use(loggedInPageStudent);
+    },
+
+    /****************************************
+     * AUTHENTICATED SESSION - DYNAMIC URLS *
```

Comment:
```suggestion
     * STUDENT - DYNAMIC URLS *
```
Some reference to these being student could be nice, as this is clearly how they are split.